fapolicy_analyzer/glade/confirm_deployment_dialog.glade

```diff
@@ -63,7 +63,22 @@
               <packing>
                 <property name="expand">True</property>
                 <property name="fill">True</property>
-                <property name="position">1</property>
+                <property name="position">2</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkButton" id="expandButton">
+                <property name="label">Expand</property>
```

Comment:
Since this only deals with Rules, should it be labeled something else? "Rule Details" perhaps?